content/pt-br/docs/reference/scheduling-policies/policies.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Políticas do escalonador
+content_type: concept
+sitemap:
+  priority: 0.2 # Scheduling priorities are deprecated
+---
+
+<!-- overview -->
+
+Nas versões anteriores à v1.23 do Kubernetes, a política do escalonador pode ser usada para especificar os processos *predicates* e *priorities*. Por exemplo, você pode definir a política de escalonador rodando os seguintes comandos:
```

Comment:
A few suggestions here:
* Perhaps write "política do escalonador" on the second sentence too.
* I would reword this to include the commands inline in the sentence like the original documentation. This code block is looking very weird in the rendered version.